server/models/user.coffee

```diff
@@ -20,3 +20,14 @@ module.exports.getDisplayName = (callback) ->
             callback null, name
         else
             callback null, null
+
+
+module.exports.getEmail = (callback) ->
+    cozydb.api.getCozyUser (err, user) ->
+        if user?
+            callback null, user.email
+        else
+            # FIXME: Why the hell do we returns err == null if we cannot get a
```

Comment:
Please no discussion in the comments. If you are not satisfied with the CozyDb API you can open an issue on the related repository.